site/components/pages/Retirements/SingleRetirement/DownloadCertificateButton/index.tsx

```diff
@@ -30,11 +29,13 @@ export const DownloadCertificateButton: FC<DownloadCertificateButtonProps> = (
 
   return (
     <ButtonPrimary
+      variant="link"
       onClick={handleClick}
-      label={t({
-        id: "retirement.single.download_certificate_button",
-        message: "Download PDF",
-      })}
+      // Replace occurrences of "\" with ZERO WIDTH SPACE (U+200B) to allow word break
+      label={props.retirementUrl
+        .replace("https://", "")
+        .replace("//", "/")
+        .replace(/\//g, "/\u200B")}
```

Comment:
to @0xAeterno 's point Hyperlinking to the certificate was a mistake in the designs. Probably a remnant from way back before we ever actually implemented PDFs. I tagged @theprofessora in figma to get that updated. But lets stick with the "Download PDF" button (see prod)

I would remove the hyperlink mode from the button component as well. Links should be links, buttons should be buttons-- again it was a design mistake. Lots of design-debt in figma because of our early velocity and because theyve been passed between several designers

Reply:
Fair enough, removed "link" as a button type and reverted to `Primary`.

>The pledge checking bit doesn't seem to be working either. I might have an idea on why, I'll comment on the relevant section below.

^  I'll confirm this once it's deployed but unable to get the Pledge object when developing locally since I'm targeting my own Firestore DB rather than the one Klima is using.
cc @Atmosfearful @0xAeterno

Reply:
Strangely it seems to work in the case of:

https://klimadao-site-git-816-retirement-pledge-ctas-klimadao.vercel.app/retirements/brice.klima/1

But not for (which has a pledge though with no value):

https://klimadao-site-git-816-retirement-pledge-ctas-klimadao.vercel.app/retirements/mcollins.eth/1